changes/fix-14423.en.md

```diff
@@ -0,0 +1 @@
+Fixed the issue that the built-in SQL function `str_utf8(Data: Term) -> string` cannot correctly handle array type json objects.
```

Comment:
> cannot correctly handle **array type json objects**

Is it correct? I see the code is Unicode related instead of JSON Array.

Reply:
Rule SQL
```sql
select 
  str(payload.listobj), // work
  bstr_utf8(payload.listobj) //not work, but it should
from 
  "t/1"
```

data:
```json
{"listobj": ["hello", 1.1 , "你好👋", true]}
```